src/XMakeTasks/Resources/Strings.resx

```diff
@@ -2606,6 +2606,9 @@
   <data name="Copy.ExactlyOneTypeOfLink">
     <value>MSB3891: Both "{0}" and "{1}" were specified in the project file. Please choose one or the other.</value>    
   </data>
+  <data name="Copy.SameDestinationPath">
+    <value>MSB3892: There are multiple source files that try to write to the same destination file: "{0}</value>    
```

Comment:
I think we'll be changing this for other reasons but: missing quote and period at the end of the message.